client/state/data-layer/wpcom/sites/blog-stickers/index.js

```diff
@@ -33,6 +34,8 @@ export function receiveBlogStickerList( store, action, next, response ) {
 		receiveBlogStickerListError( store, action, next );
 		return;
 	}
+
+	store.dispatch( receiveBlogStickers( action.payload.blogId, response ) );
```

Comment:
@blowery mentioned adding a `fromApi` here but not sure where it'd fit in. The API response looks like this (no keys):

[ 'do-not-recommend', 'sticker2' ]

When there are no stickers, we get an empty array back.

If the response is empty or is not an array, we already fire `receiveBlogStickerListError()`.